dev/tests/integration/testsuite/Magento/Store/Model/CreateStoreTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Store\Model;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ * phpcs:disable Magento2.Security.Superglobal
+ */
+class CreateStoreTest extends \PHPUnit\Framework\TestCase
+{
+
+    /**
+     * @var \Magento\Framework\ObjectManagerInterface
+     */
+    protected $objectManager;
+
+    /**
+     * @var \Magento\Store\Model\CreateStore
+     */
+    protected $createStoreModel;
+
+    /**
+     * @var \Magento\Store\Model\Website
+     */
+    protected $websiteModel;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp(): void
+    {
+        $this->objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
+
+        $this->createStoreModel = $this->objectManager->get(
+            \Magento\Store\Model\CreateStore::class
+        );
+
+        $this->websiteModel = $this->objectManager->create(
+            \Magento\Store\Model\Website::class
+        );
+        $this->websiteModel->load(1);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function tearDown(): void
```

Comment:
I don't think this method is necessary. Properties are getting reinitialized in setUp executed before each test run